src/hub/swagger.yaml

```diff
@@ -0,0 +1,228 @@
+swagger: "2.0"
+
+info:
+  title: "SubHub"
```

Comment:
Is this really still a great title for this?

Reply:
open to suggestions